app/grandchallenge/algorithms/templates/algorithms/display_set_from_job_form.html

```diff
@@ -3,7 +3,7 @@
 {% load url %}
 
 {% block title %}
-    Create a Display Set - {{ object.pk }} - {{ block.super }}
+    Create a Display Set from Result {{ object.pk }} - {{ object.algorithm_image.algorithm.title }} - {{ block.super }}
```

Comment:
Here the title is different from the breadcrumbs, right? "Create Display Set" vs "Create Display Set from Result"

Reply:
True, but then we won't be able to make the titles clear as proposed in the issue earlier [here](https://github.com/comic/grand-challenge.org/issues/3556#issuecomment-2390943068)

Then, I will only fix the incorrect things and casing, this applies to the other two comments too.

Reply:
Ok, but why not change it to "Display Set from Result" in both the title and the breadcrumb?

Reply:
That is also fine. I will change it.